test/.azure-pipelines/pull_request.yml

```diff
@@ -0,0 +1,36 @@
+parameters:
+- name: verbose
+  displayName: Verbose
+  type: boolean
+  default: false
+
+trigger:
+  branches:
+    exclude:
+      - '*'
+
+pr:
+  autoCancel: true
+  branches:
+    include:
+      - '*'
+
+extends:
+  template: ../../stages.yml
+  parameters:
+    verbose: ${{ parameters.verbose }}
+    config:
+      linux:
+        py_versions: ['3.7']
```

Comment:
Do we want to see if it also works for multiple python versions?

Reply:
Wouldn't that be a test of the `yml` syntax rather than our templates?